hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/plan/generators/HoodieLogCompactionPlanGenerator.java

```diff
@@ -55,7 +54,6 @@ public HoodieLogCompactionPlanGenerator(HoodieTable table, HoodieEngineContext e
     super(table, engineContext, writeConfig, executor);
     this.compactionStrategy = HoodieCompactionStrategy.newBuilder()
         .setStrategyParams(getStrategyParams())
-        .setCompactorClassName(LogCompactionExecutionHelper.class.getName())
```

Comment:
do we also need to adjust the `HoodieCompactionStrategy` avsc file.

Reply:
The field is nullable but it probably makes more sense to keep setting this for now for compatibility between versions.

Reply:
This field is used to determine whether log compaction execution should be used by existing releases.  So we should just backfill the same class name for backwards compatibility.